lib/fastlane/plugin/get_new_build_number/actions/get_new_build_number_action.rb

```diff
@@ -60,82 +69,37 @@ def self.authors
-        # Optional:
-        ""
+        "Fetches the latest build number from multiple services and increments it by 1."
       end
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(
-            key: :bundle_identifier,
-            env_name: "APP_BUNDLE_ID",
-            description: "iOS bundle identifier",
-            optional: true,
-            type: String
-          ),
-          FastlaneCore::ConfigItem.new(
-            key: :package_name,
-            env_name: "APP_PACKAGE_NAME",
-            description: "Android package name",
-            optional: true,
-            type: String
-          ),
-          FastlaneCore::ConfigItem.new(
-            key: :google_play_json_key_path,
-            env_name: "GOOGLE_PLAY_JSON_KEY_PATH",
-            description: "Path to the Google Play Android Developer JSON key",
-            optional: true,
-            type: String
-          ),
-          FastlaneCore::ConfigItem.new(
-            key: :app_store_initial_build_number,
-            env_name: "APP_STORE_INITIAL_BUILD_NUMBER",
-            description: "Build number to use if there's nothing in App Store",
-            optional: true,
-            type: String
-          ),
-          FastlaneCore::ConfigItem.new(
-            key: :firebase_json_key_path,
-            env_name: "FIREBASE_JSON_KEY_PATH",
-            description: "Path to the Firebase Admin JSON key",
-            optional: true,
-            type: String
-          ),
-          FastlaneCore::ConfigItem.new(
-            key: :firebase_app_ios,
-            env_name: "FIREBASE_APP_IOS",
-            description: "Firebase iOS app ID",
-            optional: true,
-            type: String
-          ),
-          FastlaneCore::ConfigItem.new(
-            key: :firebase_app_android,
-            env_name: "FIREBASE_APP_ANDROID",
-            description: "Firebase Android app ID",
-            optional: true,
-            type: String
-          ),
-          FastlaneCore::ConfigItem.new(
-            key: :use_temp_build_number,
-            env_name: "USE_TEMP_BUILD_NUMBER",
-            description: "Cache the build number across multiple runs of this action",
-            optional: true,
-            type: Boolean
-          )
+          config_item(:bundle_identifier, "iOS bundle identifier"),
+          config_item(:package_name, "Android package name"),
+          config_item(:google_play_json_key_path, "Path to the Google Play Android Developer JSON key"),
+          config_item(:app_store_initial_build_number, "Build number to use if there's nothing in App Store"),
+          config_item(:firebase_json_key_path, "Path to the Firebase Admin JSON key"),
+          config_item(:firebase_app_ios, "Firebase iOS app ID"),
+          config_item(:firebase_app_android, "Firebase Android app ID"),
+          config_item(:use_temp_build_number, "Cache the build number across multiple runs of this action", type: Boolean)
```

Comment:
Thanks for this, but I prefer to keep things explicit and repeated. Also env vars – I like them explicit, not doing the to_upper_case.

So please revert the implementation of `available_options` method to what it was before.